README.md

```diff
@@ -1,372 +1,221 @@
 
-**Gem Published:** https://rubygems.org/gems/react_on_rails
+# React on Rails
+React on Rails integrates Facebook's [React](https://github.com/facebook/react) front-end framework with Rails. Currently, both React v0.14 and v0.13 are supported. See the Rails on Maui [blog post](http://www.railsonmaui.com/blog/2014/10/03/integrating-webpack-and-the-es6-transpiler-into-an-existing-rails-project/) that started it all!
 
-**Current Version:** 1.0.0.pre
+Like the [react-rails](https://github.com/reactjs/react-rails) gem, React on Rails is capable of server-side rendering with fragment caching and is compatible with [turbolinks](https://github.com/rails/turbolinks). Unlike react-rails, which depends heavily on sprockets and jquery-ujs, React on Rails uses [webpack](http://webpack.github.io/) and does not depend on jQuery. While the initial setup is slightly more involved, it allows for advanced functionality such as:
 
-**Live example, including server rendering + redux:** http://www.reactrails.com/
++ [Redux](https://github.com/rackt/redux)
++ [Webpack dev server](https://webpack.github.io/docs/webpack-dev-server.html) with [hot module replacement](https://webpack.github.io/docs/hot-module-replacement-with-webpack.html)
++ [Webpack optimization functionality](https://github.com/webpack/docs/wiki/optimization)
++ **(Coming Soon)** [React Router](https://github.com/rackt/react-router)
 
-Sponsored by [ShakaCode.com](http://www.shakacode.com/)
+See the [react-webpack-rails-tutorial](https://github.com/justin808/react-webpack-rails-tutorial/) for an example of a live implementation and code.
 
-See [Action Plan for v1.0](https://github.com/shakacode/react_on_rails/issues/1). We're ready v1.0!
+### Including your React Component in your Rails Views
+Please see [Getting Started](#getting-started) for how to set up your Rails project for React on Rails if you have not already done so.
 
-Feedback and pull-requests encouraged! Thanks in advance! We've got a private slack channel to discuss react + webpack + rails. [Email us for an invite contact@shakacode.com](mailto: contact@shakacode.com).
++ *Normal Mode (JavaScript is rendered on client):*
 
-Supports:
+  ```erb
+  <%= react_component("HelloWorldApp", { @some_props }) %>
+  ```
++ *Server-Side Rendering:*
+
+  ```erb
+  <%= react_component("HelloWorldApp", { @some_props }, prerender: true) %>
+  ```
+
++ The `component_name` parameter here would be a string matching the name you used when globally exposing your React component.
```

Comment:
@justin808 it's in "Globally Exposing Your React Components" just below, but this is why I was thinking this is coming too early since no one will yet understand how the gem works. If they're just using this for an API after already knowing the gem, then they should already understand this.